model_analyzer/plots/detailed_plot.py

```diff
@@ -168,7 +168,6 @@ def plot_data(self):
 
         # Sort the data by request rate or concurrency
         if "request_rate" in self._data and self._data["request_rate"][0]:
-            print(f"\n\nFound request rate: {self._data['request_rate']}\n\n")
```

Comment:
Trying to understand the change. Is this not needed anymore?

Reply:
That was a debug comment that somehow found got checked-in previously. Found it by inspection when making these changes.